src/bin/tikv-ctl.rs

```diff
@@ -990,6 +1050,37 @@ fn main() {
                     ),
                 ),
         )
+        .subcommand(
+            SubCommand::with_name("remove-region")
+                .about("remove a region")
+                .arg(
+                    Arg::with_name("region")
+                        .short("r")
+                        .takes_value(true)
+                        .help("the region id"),
+                ),
+        )
+        .subcommand(
+            SubCommand::with_name("init-empty-region")
+                .about("init an empty region on the store")
+                .arg(
+                    Arg::with_name("pd")
```

Comment:
Is it better to make the PD endpoints a first level argument, like the host and certification arguments?

Reply:
Most APIs do not require PD.

Reply:
So I prefer keep it as a subcommand argument.